### ISSUE TITLE
error on install

### ISSUE THREAD

**Issue by willymec** [tbar0970/jethro-pmm#525]
Getting this error at the top of the installer, while trying to install the project
![parseerror](https://user-images.githubusercontent.com/42074925/43659076-b9d41cba-970f-11e8-988e-ec49a0ec1901.png)

Works fine when I install it on my home server. Get this error only when I try to install on my website server?
Bill

**Comment by tbar0970** [comment]
Hi Bill

I'm not quite sure what the problem here is, but it looks like it's in the LESS compiler.  This only normally comes into play when you're running Jethro from the unpackaged source code, rather than a bundled release.  Which link did you use on the release page?  You'd be best off with the top one, not the source code one.
![image](https://user-images.githubusercontent.com/2133676/43696689-a106184e-9982-11e8-926c-85b47cd0a1ae.png)

**Comment by willymec** [comment]
Ok I'm pretty sure I did the top one, but will try that one for sure and see what happens.
thank you

**Comment by willymec** [comment]
Ok I reloaded with the top Jethro-2.24.0.zip
And the installer worked as designed until the login screen.
I cant login. And I know the login is correct. reloaded again to be sure.

Jethro PMM RollCall
Login Key Incorrect. Please try again.

**Comment by tbar0970** [comment]
That suggests an issue with cookies or sessions.  Can you try it in incognito mode, and make sure PHP sessions are working/enabled on your server.

**Comment by tbar0970** [comment]
Yep it sounds like a hosting issue.  I don't know freehosting.com but there could be some setting to turn on PHP sessions. (Or, if it's free hosting, it's possible they don't support PHP sessions?)

**Comment by s4069b** [comment]
I've been hacked before when using free/bargain hosting. much safer to pay for a reliable host (particularly when storing people's details). 
Or use https://easyjethro.com.au/ and let them worry about the setup!

**Comment by willymec** [comment]
My website works perfect on freehosting.com. Thought i would try this on there before i paid. Also we will not have any senstive info. Only needing to track Sunday School classes attendace. And we have 5 classes. I made an access program that worked perfect but cant work on the ipads we have. So going back to plan a and host it locally. 
Thank you
